### PR TITLE
Add tests for api/v1/conferences_controller

### PR DESCRIPTION
**Pull request by nishanthvijayan**

### CODE REVIEW

File: spec/controllers/api/v1/conferences_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+
+describe Api::V1::ConferencesController do
+  let!(:conf_one) { create(:conference, short_title: 'conf_one') }
+  let!(:conf_two) { create(:conference, short_title: 'conf_two') }
+
+  describe 'GET #index' do
+    it 'returns all conferences' do
+
+      get :index, format: :json
```
Comment (ChrisBr) [inline spec/controllers/api/v1/conferences_controller_spec.rb]: Can you move this and the next line in a before block and then make three test cases for your test?

File: spec/controllers/api/v1/conferences_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      expect(json.length).to eq(2)
+      expect(json[0]['short_title']).to eq('conf_one')
+      expect(json[1]['short_title']).to eq('conf_two')
+    end
+  end
+
+  describe 'GET #show' do
+    it 'returns a specific conference' do
+
+      get :show, id: 'conf_two', format: :json
```
Comment (ChrisBr) [inline spec/controllers/api/v1/conferences_controller_spec.rb]: Same for this one!

**Comment by ChrisBr** [comment]
Otherwise LGTM

**Comment by nishanthvijayan** [comment]
I've the made the changes suggested by @ChrisBr

**Comment by hennevogel** [comment]
:white_check_mark: lets go! Thanks  :-)

**Comment by hennevogel** [comment]
Then let's go :-)

**Pull request merged**